hack/flux/update-flux.sh

```diff
@@ -45,24 +45,16 @@ function update_flux() {
     kustomize create --autodetect
     popd && popd
 
-    # Update flux version in defaultApps whenever flux version is upgraded.
-    sed -i "s/kommander-flux: \".*\"/kommander-flux: \"$LATEST_FLUX_VERSION\"/g" services/kommander/*/defaults/cm.yaml
-
     git add services
 
-    if [[ -z "$(git config user.email 2>/dev/null || true)" ]]; then
-        git config user.email "ci@mesosphere.com"
-        git config user.name "mesosphere-teamcity"
-    fi
-
     readonly COMMIT_MSG="feat: Upgrade flux to ${LATEST_FLUX_VERSION}"
 
     git commit -m "${COMMIT_MSG}"
 
     git push --set-upstream origin "${BRANCH_NAME}"
 
     git fetch origin main
-    KOMMANDER_APPLICATIONS_PR=$(gh pr create --base main --fill --head "${BRANCH_NAME}" -t "${COMMIT_MSG}" -l ready-for-review -l ok-to-test -l slack-notify -l open-kommander-pr -l update-licenses)
+    KOMMANDER_APPLICATIONS_PR=$(gh pr create --base main --fill --head "${BRANCH_NAME}" -t "${COMMIT_MSG}" -l ready-for-review -l ok-to-test -l slack-notify -l update-licenses)
```

Comment:
Is the slack notify label still working? Either way we can drop the ok to test label and perhaps add test/kuttl label

Reply:
these are labels for the k-apps PR I believe -- for the kommander PR I added all the `test/*` labels. I don't think slack-notify works but maybe we should fix that eventually so I'll just leave it here until then 😬